avocado/utils/multipath.py

```diff
@@ -99,7 +99,7 @@ def get_multipath_wwids():
     """
     cmd = "egrep -v '^($|#)' /etc/multipath/wwids"
     wwids = process.run(cmd, ignore_status=True,
-                        sudo=True).stdout_text
```

Comment:
@smruti77  I don't follow the reason for this change.  For instance:

```
$ python3
>>> cmd = "egrep -v '^($|#)' /etc/multipath/wwids"


>>> process.run(cmd)
<avocado.utils.process.CmdResult object at 0x7faa506ca950>
>>> process.run(cmd).stdout_text
'/0QEMU_QEMU_HARDDISK_3600508b400105e21000/\n'



>>> process.run(cmd, shell=True)
<avocado.utils.process.CmdResult object at 0x7faa506d5490>
>>> process.run(cmd, shell=True).stdout_text
'/0QEMU_QEMU_HARDDISK_3600508b400105e21000/\n'
```

I don't see why a shell is needed here and in all the commands I tried.  Do you have a reproducer?  Or maybe the reviewers @beraldoleal @PraveenPenguin can help me with that too.

Reply:
Yes, @clebergnu I see your point and I agree with it. Maybe just a try... except will be enough here.

@smruti77, do you have the exact error that you are getting?